server/config.ts

```diff
@@ -6,6 +6,11 @@ import configDefaults from "./configDefaults";
 
 export default configDefaults;
 
+/* tslint:disable */
+let bcrypt = require("bcryptjs");
+/* tslin:enable */
+let bcryptRegex = /^\$2a\$10\$/;

```

Comment:
I think the following should do it: `/^\$2a\$10\$.{54}/`, [interpreted by Regexper](http://regexper.com/#%2F%5E%5C%242a%5C%2410%5C%24.%7B54%7D%2F).